apps/openmw/mwclass/container.cpp

```diff
@@ -321,7 +321,8 @@ namespace MWClass
 
     bool Container::canLock(const MWWorld::ConstPtr &ptr) const
     {
-        return true;
+        const MWWorld::LiveCellRef<ESM::Container> *ref = ptr.get<ESM::Container>();
+        return !(ref->mBase->mFlags & ESM::Container::Organic);
```

Comment:
Actually, organic containers can be locked in Morrowind. It just is not possible to lock them via spell, so you should add check to the lock effect handling instead of `canLock`.

Reply:
canLock isn't checked for scripted locking/unlocking (it just throws if the object physically can't be locked), so technically this is valid, especially since with the check removed for pickLock the only place where canLock is used is spellcasting. Though I'm not sure why *should* it be removed for lockpicking.

Reply:
Uhm... guess what? Morrowind allows lockpicking on activators. Yay?

Reply:
Hm, okay then.